server/handlers/default.go

```diff
@@ -20,6 +25,14 @@ import (
 	"github.com/docker/notary/tuf/validation"
 )
 
+const (
+	// ConsistentCacheMaxAge is the Cache-Control header's max age for consistent downloads
+	ConsistentCacheMaxAge int = 30 * 24 * 60 * 60 // 30 days
+
+	// NonConsistentCacheMaxAge is the Cache-Control header's max age for current (non-consistent) downloads
+	NonConsistentCacheMaxAge int = 5 * 60 // five minutes
```

Comment:
IRL discussion with @cyli: it seems that making these cache times configurable would also be nice